fiftyone/utils/qwen3_vl.py

```
@@ -403,3 +411,149 @@ def embed_all(self, args):
+
+        Returns:
+            a 1D numpy array embedding
+        """
+        from PIL import Image as PILImage
+
+        total_frames = video_reader.total_frame_count
+        raw_fps = video_reader.frame_rate
+        sample_fps = self.config.video_fps
+
+        if raw_fps > 0 and sample_fps > 0:
+            step = max(1, int(round(raw_fps / sample_fps)))
```

Ruff (0.15.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)